frontend/src/component/ui/ChangeLogModal.scss

```
@@ -0,0 +1,113 @@
+@import '../styles/color';
+@import '../styles/corner';
+@import '../styles/scrollbar';
+
+.modal-wrapper {
+  left: 0%;
+  position: fixed;
+  top: 0%;
+  width: 100%;
+  height: 100%;
+
+  .modal-body {
+    background-color: $color-white;
+    border: 1px solid $color-light-gray-1;
+    left: 50%;
+    position: fixed;
+    top: 50%;
+    width: 500px;
+    height: 500px;
+    transform: translateX(-50%) translateY(-50%);
+    @include round-corner;
+
+    .modal-header {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      font-size: 20px;
+      line-height: 1;
+      text-align: left;
+      padding: 20px;
+      border-bottom: 1px solid $color-light-gray-1;
+      font-weight: 600;
+
+      i.clear {
+        cursor: pointer;
+      }
+    }
+
+    ul {
+      height: calc(500px - 114px);
+      list-style: none;
+      display: block;
+      padding: 0;
+      margin: 0;
+      overflow-y: scroll;
+      background-color: $color-white-smoke;
+      border: 4px solid $color-white-smoke;
+      @include scroll-bar;
+
+      li {
+        padding: 10px;
+        list-style: none;
+        text-align: left;
+        border: 1px solid $color-light-gray-1;
+        margin: 10px;
+        min-height: 120px;
+        background: white;
+        position: relative;
+        word-break: break-all;
+
+        .title {
+          font-weight: 900;
+          padding-bottom: 6px;
```

```suggestion
          margin-bottom: 6px;
```
Padding counts as part of the height of title. I guess you want to keep distance between title and description. Please checkout CSS box modal: https://www.w3schools.com/css/css_boxmodel.asp